tests/integration/sequential/test_par.py

```diff
@@ -385,3 +385,20 @@ def test_par_sequence_index_is_numerical():
     s1.fit(data)
     sample = s1.sample(2, 5)
     assert sample.columns.to_list() == data.columns.to_list()
+
+
+def test_par_error_on_context_columns():
+    metadata_dict = {
```

Comment:
Can you add docstring for this test and the unit test along with sectioning the test code with:
`#Setup`, `#Run`, `#Assert`

Reply:
Added:
https://github.com/sdv-dev/SDV/pull/2108/commits/ab3c87b279dc66bbd177cb6803c5881f748dd745